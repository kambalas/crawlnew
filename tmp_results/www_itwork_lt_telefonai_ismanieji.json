[
  {
    "name": "Google Pixel 9 Pro 128GB 16RAM 5G Obsidian",
    "price": "1078.35",
    "error": false
  },
  {
    "name": "Google Pixel 9 16 cm (6.3), 12 GB, 128 GB, 50 MP, Android 14, Wintergreen",
    "price": "904.68",
    "error": false
  },
  {
    "name": "Google Pixel 8a 128GB 8RAM 5G bay",
    "price": "479.49",
    "error": false
  },
  {
    "name": "Google Pixel 7 256GB 8RAM 5G white",
    "price": "468.90",
    "error": false
  },
  {
    "name": "Smartfon Google Pixel 9 5G Dual Sim 12GB RAM 128GB - Obsidian",
    "price": "712.15",
    "error": false
  },
  {
    "name": "MOBILE PHONE PIXEL 9 128GB/PEONY GA05841-GB GOOGLE",
    "price": "726.30",
    "error": false
  },
  {
    "name": "Google Pixel 9 Pro 128GB 16RAM 5G Rose",
    "price": "918.79",
    "error": false
  },
  {
    "name": "MOBILE PHONE PIXEL 9A 5G 256GB/OBSIDIAN GA09566-GB GOOGLE",
    "price": "714.00",
    "error": false
  },
  {
    "name": "MOBILE PHONE PIXEL 9 PRO/16/128 HAZEL GA05936-GB GOOGLE",
    "price": "923.08",
    "error": false
  },
  {
    "name": "MOBILE PHONE PIXEL 9 PRO XL/128GB OBISID GA05360-GB GOOGLE",
    "price": "935.84",
    "error": false
  },
  {
    "name": "MOBILE PHONE PIXEL 9 256GB/PORCELAIN GA05843-GB GOOGLE",
    "price": "819.69",
    "error": false
  },
  {
    "name": "MOBILE PHONE PIXEL 8A 256GB/OBSIDIAN GA05571-GB GOOGLE",
    "price": "577.21",
    "error": false
  },
  {
    "name": "google | Pixel 6 Pro | Stormy Black | 6.71 | LTPO AMOLED | 1440 x 3120 | Google Tensor | Internal RAM 12 GB | 128 GB | Nano-SIM | 3G | 4G | 5G | Main camera resolution 50 + 48 + 12 MP | Secondary camera resolution 11.1 MP | Android | 12 | 5003 mAh",
    "price": "408.22",
    "error": false
  },
  {
    "name": "Google Pixel 9a 16 cm (6.3) Dual SIM Android 15 5G USB Type-C 8 GB 128 GB 5100 mAh Rose",
    "price": "610.23",
    "error": false
  },
  {
    "name": "Google Pixel 9a 16 cm (6.3) Dual SIM Android 15 5G USB Type-C 8 GB 128 GB 5100 mAh Black",
    "price": "567.74",
    "error": false
  },
  {
    "name": "Google Pixel 9 Pro Fold 256GB 16RAM 5G porcelain",
    "price": "2,304.14",
    "error": false
  },
  {
    "name": "Google Pixel 9 Pro 16 cm (6.3), 16 GB, 256 GB, 50 MP, Android 14, Hazel",
    "price": "1,078.35",
    "error": false
  },
  {
    "name": "Google Pixel 9 Pro 128GB 16RAM 5G Obsidian",
    "price": "904.68",
    "error": false
  },
  {
    "name": "Google Pixel 9 16 cm (6.3), 12 GB, 128 GB, 50 MP, Android 14, Wintergreen",
    "price": "",
    "error": false
  }
]