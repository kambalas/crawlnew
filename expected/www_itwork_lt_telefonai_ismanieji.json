[
  {
    "name": "Google Pixel 8a 128GB 8RAM 5G bay",
    "price": "479",
    "error": false
  },
  {
    "name": "Google Pixel 7 256GB 8RAM 5G white",
    "price": "468.90",
    "error": false
  },
  {
    "name": "Google Pixel 7 128GB 8RAM 5G white",
    "price": "432.48",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: TKOGOGSZA0039",
    "price": "722.12",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: GA05841-GB",
    "price": "735.16",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 3 Modelis: GA05941-GB",
    "price": "918.79",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: GA09566-GB",
    "price": "714.00",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 3 Modelis: GA05843-GB",
    "price": "848.00",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: GA05936-GB",
    "price": "935.73",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: GA05360-GB",
    "price": "959.14",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 1 Modelis: GA05227-GB",
    "price": "922.70",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: GA05571-GB",
    "price": "577.21",
    "error": false
  },
  {
    "name": "Moki 10 Gamintojas: GOOGLE Likutis: 5+ Modelis: GA04538-GB",
    "price": "468.90",
    "error": false
  }
]